src/me/components/DocSearchWidget.tsx

```diff
@@ -30,7 +30,7 @@ const DocSearchWidget: FC = () => {
   return (
     <div className="WidgetSearch">
       <DocSearch />
-      <p className="WidgetHelperText">Press CTRL + h on any page to search</p>
+      <p className="WidgetHelperText">Press CTRL + M on any page to search</p>
```

Comment:
not sure if it makes a difference but `m` vs `M`?

Reply:
Yeah, @mavarius specified it should be an uppercase M